src/eos_parse_eosio.c

```diff
@@ -176,7 +180,9 @@ void parseUpdateAuth(uint8_t *buffer, uint32_t bufferLength, uint8_t argNum, act
         return;
     }
 
-    uint8_t *keys = buffer += 3 * sizeof(name_t) + sizeof(uint32_t);
+    // moved buffer build before keys assigment, previously inline and dead-code
+    buffer += 3 * sizeof(name_t) + sizeof(uint32_t);
+    uint8_t *keys = buffer;
     uint32_t keyBufferLength = bufferLength - 3 * sizeof(name_t) + sizeof(uint32_t);
```

Comment:
I see a bug here (that was there before). Line should be:
`uint32_t keyBufferLength = bufferLength - (3 * sizeof(name_t) + sizeof(uint32_t));

It might be a bit clearer to do this:

```
    uint32_t prefixLength = 3 * sizeof(name_t) + sizeof(uint32_t);
    buffer += prefixLength;
    uint8_t *keys = buffer;
    uint32_t keyBufferLength = bufferLength - prefixLength ;
```